File: ABAC/backend/backend/Entities/Team.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Entities
{
    public class Team
    {
        [Key]
        public int Id { get; set; }
        public Captain Captain { get; set; }
        public List<Robot> Robots { get; set; } = new List<Robot> { };
    }
}

File: ABAC/frontend/src/app/models/team.ts
import Captain from './captain';
import Robot from './robot';

export default class Team {
  public id: number;
  public captain: Captain;
  public robots: Robot[];

  constructor(id: number, captain: Captain, robots: Robot[]) {
    this.id = id;
    this.captain = captain;
    this.robots = robots;
  }
}
